validator/accounts/v2/accounts_deposit.go

```diff
@@ -137,6 +137,20 @@ func createDepositConfig(cliCtx *cli.Context, km *derived.Keymanager) (*derived.
 				return nil, err
 			}
 			config.Eth1PrivateKey = strings.TrimRight(string(fileBytes), "\r\n")
+		} else {
+			config.Eth1KeystoreUTCFile = cliCtx.String(flags.Eth1KeystoreUTCPathFlag.Name)
```

Comment:
Can we please test this logic? Thanks!

Reply:
Working on it.  Thanks

Reply:
Addressed.  Can you take a look at the tests?